manylinux2014-wheel-build/Makefile

```diff
@@ -3,7 +3,7 @@ TARGET := $(notdir $(WD))
 ROOT := $(abspath $(WD)/../Pillow)
 IMAGENAME := $(if $(DOCKER_USERNAME), $(DOCKER_USERNAME)/$(TARGET), $(TARGET))
 BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
-TEST_TARGET = ubuntu-24.04-noble-amd64
+TEST_TARGET = fedora-41-amd64
```

Comment:
Because Fedora 41 comes with Python 3.13, and Ubuntu 24.04 comes with 3.12?

I've not been using these manylinux images locally, what's the other pros/cons of switching?

Does it affect the main Pillow CI?

Reply:
Yes, the Python version is why.

I don't expect there to be any side effects of changing the test image - it won't affect the build itself. As the [README](https://github.com/python-pillow/docker-images/blob/main/manylinux_2_28-wheel-build/README.md) says
> The test target here is mainly to validate the image build, it is assumed that the builds will be used for other purposes or tested in other images.

The wheel build jobs aren't used by the main repository.

Reply:
Thanks!